nepalingo-web/README.md

```diff
@@ -0,0 +1,65 @@
+# nepalingo
+
+A website to learn the indigenous language of Nepal. This project is a React application built with TypeScript, Vite, and Tailwind CSS. Below are the steps to set up and run the project.
+
+## Prerequisites
+
+Ensure you have the following installed on your machine:
+
+- [Node.js](https://nodejs.org/en/download/)
+- [pnpm](https://pnpm.io/installation)
+- [Visual Studio Code](https://code.visualstudio.com/) (recommended)
+
+## Installation
+
+1. **Clone the repository:**
+
+    ```sh
+    git clone https://github.com/nepalcodes/nepalingo.git
+    cd nepalingo-web
+    ```
+
+2. **Install pnpm (if you have not done it already)**
+   ```sh
+   npm install -g pnpm
+   ```
+3. **Install dependencies:**
+
+    ```sh
+    pnpm install
+    ```
+
+
+## Running the Development Server
+
+To start the development server, run the following command:
+
+```sh
+pnpm run dev
+```
+
+## Project Structure
+my-react-app/
+├── public/
+│   ├── index.html
+│   └── ...
+├── src/
+│   ├── assets/
+│   │   ├── images/
+│   │   └── styles/
```

Comment:
Let's put styles next to the components. You can remove this folder

Reply:
Why is this important?

Reply:
This makes it so that when you update a component you don't need to go hunting in the folder structure for its styles - they are right there next to it